### PR TITLE
Use busybox from quay.io

### PR DESCRIPTION
**Pull request by jiripetrlik**
Use busybox from quay.io instead of from Docker hub. Version should stay the same.

### CODE REVIEW

File: tests/integration/ray_test.go [hunk 1]
```diff
@@ -319,7 +319,7 @@ func createRayCluster(test support.Test, namespace string, mnist *corev1.ConfigM
 							InitContainers: []corev1.Container{
 								{
 									Name:    "init-myservice",
-									Image:   "busybox:1.28",
+									Image:   "quay.io/project-codeflare/busybox:1.28",
```
Comment (dimakis) [inline tests/integration/ray_test.go:322]: Is there a reason we're using `1.28` and not `1.36` which appears to be the latest one we've got in quay?

Comment (dimakis) [reply tests/integration/ray_test.go:322]: (quay page)[https://quay.io/repository/project-codeflare/busybox?tab=tags]

Comment (jiripetrlik) [reply tests/integration/ray_test.go]: @dimakis Yes, true you are right. Updated to 1.36.

**Comment by jiripetrlik** [comment]
Replaced by https://github.com/opendatahub-io/distributed-workloads/pull/153

**Pull request closed without merging**